src/app/shared/components/workshop-modal/workshop-modal.component.html

```diff
@@ -0,0 +1,32 @@
+<div
+    class="workshop-modal__content"
+    mat-dialog-content
+    fxLayoutAlign="center"
+  >
+    <h2 *ngIf="data.title" class="workshop-modal__title">
+      {{data?.title}}
+    </h2>
+    <p *ngIf="data.message" class="workshop-modal__message">
+      {{data?.message}}
+    </p>
+  </div>
+  <div mat-dialog-actions fxLayoutAlign="center">
+    <button
+      class="workshop-modal__action-button white"
+      mat-raised-button
+      mat-dialog-close
+    >
+      Повернутись
+    </button>
+    <button
+      *ngIf="data.buttonLabel"
+      class="workshop-modal__action-button"
+      mat-raised-button
```

Comment:
fix styles please, I belive you should add shared button styles to the scss file
![image](https://user-images.githubusercontent.com/27493959/200001769-bfe619be-7f3a-45ac-8f3e-aa5e8c55ff93.png)
besides color, both button should have teh same width

Reply:
it's not a styling issue, it's how autofocus works for modal, I'll add a property to prevent the buttons from being highlighted